cylc/flow/network/schema.py

```diff
@@ -625,11 +622,20 @@ class Meta:
 
 
 # The run mode for the workflow.
-WorkflowRunMode = graphene.Enum(
-    'WorkflowRunMode',
-    [(m.capitalize(), m) for m in WORKFLOW_RUN_MODES],
-    description=lambda x: RunMode(x.value).describe() if x else None,
-)
+class WorkflowRunMode(graphene.Enum):
+    """The mode used to run a workflow."""
+
+    # NOTE: using a different enum because:
```

Comment:
Using a different enum to what?

Can we not link this to `cylc.flow.run_modes.__init__` so that we have a single source of truth for which run modes are allowed?

Reply:
This is a regression - reverted in https://github.com/dwsutherland/cylc-flow/pull/22/files#diff-97f4d62be41218e2d70a203edfca6d44d7c0046b6908cefa8e3af7c02cd31920R617

Reply:
Doc comment is minor compared to sorting out what I'm failing to understand about the enum change - but can we say that explictly?

Reply:
(The comment is moot, it pre-dates your changes in #6039, which were accidentally reverted here. Re-reverted in my PR https://github.com/dwsutherland/cylc-flow/pull/22)

Reply:
@MetRonnie, is this resolved now?

Reply:
It's @wxtim's comment, but got to be resolved as the code the comment is on has been reverted